### ISSUE TITLE
graphql-ruby-client js dependency on graphql 16

### ISSUE THREAD

**Issue by KuIvan** [rmosolgo/graphql-ruby#4008]
Hi, @rmosolgo we use latest version of graphql package, but due to peer dependencies version lock on graphql-ruby-client js we cannot install graphql-ruby-client package without --force flag.

```
package.json
"graphql": "^16.3.0",
```

```
Could not resolve dependency:
 peer graphql@"^14.3.1 || ^15.0.0" from graphql-ruby-client@1.10.6
 node_modules/graphql-ruby-client
  graphql-ruby-client@"*" from the root project
  Fix the upstream dependency conflict, or retry
  this command with --force, or --legacy-peer-deps
  to accept an incorrect (and potentially broken) dependency resolution.
```

Is there a chance you can update version on js client?
Thanks.

**Comment by masterkain** [comment]
I too can use an upgrade, thanks

```
npm ERR! Found: graphql@16.3.0
npm ERR! node_modules/graphql
npm ERR!   graphql@"^16.3.0" from the root project
npm ERR!   peer graphql@"^14.0.0 || ^15.0.0 || ^16.0.0" from @apollo/client@3.5.10
npm ERR!   node_modules/@apollo/client
npm ERR!     @apollo/client@"^3.5.10" from the root project
npm ERR!     peer @apollo/client@"^3.3.6" from graphql-ruby-client@1.10.6
npm ERR!     node_modules/graphql-ruby-client
npm ERR!       graphql-ruby-client@"^1.10.6" from the root project
npm ERR!   2 more (graphql-ws, subscriptions-transport-ws)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer graphql@"^14.3.1 || ^15.0.0" from graphql-ruby-client@1.10.6
npm ERR! node_modules/graphql-ruby-client
npm ERR!   graphql-ruby-client@"^1.10.6" from the root project
```

```
    "@apollo/client": "^3.5.10",
    "graphql": "^16.3.0",
    "graphql-ruby-client": "^1.10.6",
    "graphql-tag": "^2.12.6",
```

**Comment by rmosolgo** [comment]
Hi! Thanks for mentioning it. I just released graphql-ruby-client 1.10.7 with looser versions on GraphQL and Apollo Client. Please let me know if you have any more trouble with it!
